tools/setup_perf_cluster.sh

```diff
@@ -134,6 +134,11 @@ function install_istio() {
   Execute sh -c 'sed -e "s/_debug//g" install/kubernetes/istio-auth.yaml | egrep -v -e "- (-v|\"2\")" | kubectl apply -f -'
 }
 
+function install_istio_addons() {
```

Comment:
This function isn't called, are you sure you meant to add it?

Reply:
yes that's the wip part, I will add it to setup all but need to manually test first, if you have time you could get that branch and give it a shot

Reply:
fixed now